changelog.d/14917.misc

```diff
@@ -0,0 +1 @@
+Improve performance of looking up partial-state status.
```

Comment:
We also have the concept of partial state events, so we may want to be a bit more specific.
```suggestion
Faster joins: Improve performance of looking up partial-state status of rooms.
```